scene/gui/popup.cpp

```diff
@@ -141,7 +141,9 @@ void Popup::popup_centered(const Size2 &p_size) {
 	rect.size = p_size == Size2() ? get_size() : p_size;
 	rect.position = ((window_size - rect.size) / 2.0).floor();
 
+	centered = true;
```

Comment:
That seems hacky, wouldn't it be better to add a boolean argument to `popup()`?

To avoid exposing it to the scripting API, make it `_popup(const Rect2 &p_bounds, bool p_centered)` and have `popup(const Rect2 &p_bounds)` call `_popup(p_bounds, false)`.

BTW, if you can take the chance to amend the commit log to use "centering" instead of "centralizing" :)

Reply:
Changed :+1: